actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -693,6 +693,22 @@ def transform_keys!(&block)
       self
     end
 
+    # Returns a new <tt>ActionController::Parameters</tt> instance with the
+    # results of running +block+ once for every key. This includes the keys
+    # from the root hash and from all nested hashes. The values are unchanged.
```

Comment:
The documentation for `deep_transform_keys` mentions nested hashes **and arrays**:

https://github.com/rails/rails/blob/bbd065204ed45071f048badb7c0d49e35773d788/activesupport/lib/active_support/core_ext/hash/keys.rb#L58-L59

Could you include that here too? Obviously arrays don't have keys, but the point is that the method traverses nested arrays of hashes, not just nested hashes. If you can think of a clearer way to explain that, all the better 😊

Reply:
Maybe something like "_This method includes the keys of nested hashes and array of hashes_"?